xml/System/Int32.xml

```diff
@@ -2423,6 +2423,8 @@
 |*e*|The 'e' or 'E' character, which indicates that the value is represented in exponential notation. The `s` parameter can represent a number in exponential notation if `style` includes the <xref:System.Globalization.NumberStyles.AllowExponent?displayProperty=nameWithType> flag.|  
 |*hexdigits*|A sequence of hexadecimal digits from 0 through f, or 0 through F.|  
   
+ Note that, for compatibility reasons, a single trailing NUL (U+0000) character is always permitted in `s`, regardless of the value of `style`.
```

Comment:
I think that I'd do this as a note and phrase it somewhat differently:
\> [!NOTE]
\> Any terminating NUL (U+0000) characters in \`s\` are ignored by the parsing operation regardless of the value of the \`style\` argument.